docs/CANDIDATE_TECHNOLOGIES.md

```diff
@@ -0,0 +1,159 @@
+- Stable and long-term support.
+- Based on JavaScript and can support Infusion.
+- Provides various Chrome APIs which would be useful in implementing certain navigation effects/features.
+- Lightweight and easy for users to set-up (and install).
+- Requires no separate execution of scripts by the user.
+- Can run on both Desktop and Mobile (supported for a few browsers only).
+
+#### Disadvantages
+
+- Cross-browser restrictions, i.e., a Chrome-based extension can't run on Firefox or vice-versa.
+
+Most of the browsers (Chrome, Microsoft Edge, Opera, etc.) are based on Chromium and can thus run Chrome Extension. The Chromium-based browsers for mobile such as [Kiwi](https://play.google.com/store/apps/details?id=com.kiwibrowser.browser&hl=en_IN) also support and allow users to install and run Chrome Extensions (unlike the Google Chrome for mobile).
+
+Although a Firefox add-on would offer the same support and features, it will only be limited to Firefox. Talking about Safari add-ons, it would be accessible only to Apple users, as Safari runs only on macOS and iOS.
+
+A Chrome extension would be accessible to a wide range of users, and so I believe that a Chrome Extension would be better than a Firefox or Safari add-on if we choose to proceed with this option. The Chrome Extension can be ported to Firefox and Safari later in the future.
+
+### Electron.js Desktop App
+
+This would involve making a desktop app that can work at the OS level.
+
+#### Advantages
+
+- Stable and long-term support.
+- OS-level execution of actions.
+- Simulate keyboard as well as mouse events.
+- Based on JavaScript and can support Infusion.
+- Easy for users to set-up (and install).
+- Requires no separate execution of scripts by the user.
+
+#### Disadvantages
+
+- No support for devices other than a desktop or a PC.
+- ([Node Key Sender](https://www.npmjs.com/package/node-key-sender)) Requires a separate Java Environment in the OS to work.
+- ([Robot.js](https://www.npmjs.com/package/robotjs)) No stable release yet.
+- Large size (> 100mb) and resource hogging.
+
+This solution would offer more control to the users in terms of navigation. The significant limitation would be that it won't work on mobile and will consume a high amount of resources. Also, a few solutions already exist which offer the users to navigate at OS-level using a gamepad.
+
+### Espruino Pico
+
+This is a bit of a different approach. As of now, we want to have browser navigation using a gamepad. But we can achieve similar results using an Internet of Things (IoT) board. Espruino Pico is one such IoT board that can be used both as a keyboard and a mouse.
+
+#### Advantages
+
+- Based on JavaScript.
+- OS-level execution of actions.
+- Simulate keyboard as well as mouse events.
+- Requires no separate execution of scripts by the user.
+- Low cost (under $50).
+
+#### Disadvantages
+
+- Requires a lot of technical setup and integration by the user.
+- Cannot use Infusion due to memory limitations.
```

Comment:
You don't have to spell it out in the document, but if you could comment with more details (source links) for the memory requirements and limitations, I'd appreciate it.  I have discussed working with Infusion on the Espruino with various community members, and it would be good to know what the issues might be.

Reply:
Hi @the-t-in-rtf, I meant that the Infusion's minified file is itself about **860 kB** while the Espruino Pico's storage capacity is about **384 kB**. I have updated this in my previous commit at [line 112](https://github.com/fluid-lab/gamepad-navigator/pull/4/files#diff-b53d05f39c28591f46c423b3aa9c65b5R112).